plansys2_executor/include/plansys2_executor/bt_builder_plugins/stn_bt_builder.hpp

```diff
@@ -15,6 +15,8 @@
 #ifndef PLANSYS2_EXECUTOR__BT_BUILDER_PLUGINS__STN_BT_BUILDER_HPP_
 #define PLANSYS2_EXECUTOR__BT_BUILDER_PLUGINS__STN_BT_BUILDER_HPP_
 
+#include <eigen3/Eigen/Dense>
```

Comment:
Is this dependency included in the `package.xml`?

Reply:
I added the Eigen3 dependency to CMakeLists.txt and package.xml. See 6119d12